tests/foreman/cli/test_host.py

```diff
@@ -2076,6 +2078,155 @@ def test_negative_multi_cv_registration(
     ), f'Expecting error "Registering to multiple environments is not enabled"; instead got: {res.stderr}'
 
 
+@pytest.mark.rhel_ver_match('[^7]')
+def test_positive_multi_cv_registration(
+    module_org,
+    module_activation_key,
+    module_lce,
+    module_lce_library,
+    module_published_cv,
+    module_promoted_cv,
+    target_sat,
+    rhel_contenthost,
+):
+    """Register a host to multiple content view environments.
+
+    :id: d0f78923-cace-4dc4-9936-81fe7e189a6b
+
+    :steps:
+        1. Register a host with global reg, just to get the sub-man config and certs right
+        2. Unregister the host
+        3. Expose the allow_multiple_content_views setting
+        4. Set the allow_multiple_content_views setting to true
+        5. Attempt to register the host with subscription-manager, passing multiple environments
+        6. Confirm that registration succeeds
+        7. Confirm that the host is registered to both environments
+
+    :expectedresults: Registration succeeds and the host is registered to both environments.
+
+    :CaseImportance: Critical
+
+    :parametrized: no
```

Comment:
vote yes here from discussion in above test, using `rhel_ver_match` to parameterize `rhel_contenthost`.